.changeset/khaki-eggs-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
+---
+
+Add polarisSummerEditions2023 feature flag to AppProvider context.
```

Comment:
Should we create multiple changesets for all the changes here? or one large one?